pymongo/command_cursor.py

```diff
@@ -158,7 +178,7 @@ def _maybe_pin_connection(self, sock_info):
             else:
                 self.__sock_mgr = sock_mgr
 
-    def __send_message(self, operation):
+    def __send_message(self, operation: Union[_Query, _GetMore]) -> None:
```

Comment:
operation can only be _GetMore here, not _Query.

Reply:
thanks for catching that!